tests/unit_tests/test_qmpc.py

```diff
@@ -154,6 +155,32 @@ def test_predict_errorhandring(self, run_server1,
                 model_param_job_uuid, model_id,
                 [["id1", "id2"], [3], [1]], [0, 1])
 
+    def test_exception_job_error(self):
+        # QMPCJobErrorとして例外がthrowされるかのテスト
+        qmpc: QMPCServer = QMPCServer(
+            ["http://localhost:9001",
+             "http://localhost:9002",
+             "http://localhost:9003"],
+            "QMPCJobError"
+        )
+        with pytest.raises(QMPCJobError):
+            qmpc.execute_computation(
+                1,
+                [["id1", "id2"], [0], [1, 1]], [[0, 1], []])
```

Comment:
Are these argument required for this test?  
In order to improve readability, I would be appreciated if it was a minimal code that could be reproduced for this behavior.

Reply:
@arukuka 
Thanks for the review
I fixed it with [e93e65e](https://github.com/acompany-develop/QuickMPC-libClient-py/pull/24/commits/e93e65e41fd29c00190add29167f6f8a54e4ef0d).